study.md

```diff
@@ -29,21 +29,23 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is code being processed.  request:response request:response
```

Comment:
The event loop is where asynchronous functions wait while the call stack is full. When the call stack is full async code lines up in the queue and when theres a chance the event loop will take the next thing from the queue and move it to the call stack.